src/Modules/Capacities/Commands.Capacities/GetPowerBICapacity.cs

```diff
@@ -0,0 +1,82 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api.Capacities;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Capacities
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(IEnumerable<Capacity>))]
+    public class GetPowerBICapacity : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBICapacity";
+        public const string CmdletVerb = VerbsCommon.Get;
+        private const string ExpandVariable = "GetPowerBICapacityExpandVariable";
+        private const string AdminVariable = "GetPowerBICapacityAdminVariable";
+
+        public GetPowerBICapacity() : base() { }
+
+        public GetPowerBICapacity(IPowerBIClientCmdletInitFactory init) : base(init) { }
+
+        #region Parameters
+
+        [Parameter(Mandatory = false)]
+        public PowerBIUserScope Scope { get; set; } = PowerBIUserScope.Individual;
+
+        [Parameter(Mandatory = false)]
+        public PowerBIGetCapacityExpandEnum Show { get; set; } = PowerBIGetCapacityExpandEnum.None;
```

Comment:
One issue with this approach is that it's not flexible for expanding multiple properties. You should reconsider the input to this parameter.